organizations/organizations.shacl.ttl

```diff
@@ -539,6 +539,22 @@
     sh:description "L'adresse de l'emplacement."@fr ;
 
     sh:message "occurs more than once or its object is no schema:PostalAddress"@en ;
+  ],
+  [
+    a sh:PropertyShape ;
+    sh:path org:siteOf ;
+    sh:maxCount 1 ;
```

Comment:
hier zouden we best een `sh:minCount` hebben om geen Vestigingen zonder organisatie te hebben

Reply:
De requested change is toegevoegd in dit datamodel. De aanpassing is ook doorgevoerd in de TriplyETL.